spec/factories/news/item_factory.rb

```diff
@@ -74,5 +74,33 @@
         CONTENT
       end
     end
+
+    trait :with_unsafe_html do
+      unsafe_html = "<script>alert('Hello, world!')</script>"
```

Comment:
This isn't really needed and tbh I'm not entirely sure the scope these variables are getting set in - I think its fine to duplicate this short string in the content and title attributes